### PR TITLE
Add CoC and CONTRIBUTING files

### PR DESCRIPTION
**Pull request by vrom911**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    git clone git@github.com:your-username/project-name.git
+
+Make your changes and consider the following checklist to go through
+before submitting your pull request.
+
+### :white_check_mark: Check list
+
+- [ ] New/fixed features work as expected (Bonus points for the new tests).
+- [ ] There are no warnings during compilation.
+- [ ] `hlint .` output is: _No Hints_ (see [`hlint`][hlint] tool docs).
```
Comment (chshersh) [inline CONTRIBUTING.md:34]: Hehe, in some future we will change this line to `PR approved by hintman` :smirk:

**Review by chshersh — APPROVED** [review]
Looking good!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+We are delighted you're reading this, and we appreciate the effort you're
+taking to make our projects awesome! :sparkles:
+
+## How to contribute
+
+### :bug: Report bugs or feature request :bulb:
+
+If you discover a bug or have any proposals on how to make this project better
+don't hesitate to create an issue [here](../../issues/new) in a free format.
```
Comment (vrom911) [inline CONTRIBUTING.md:13]: I'm afraid, such links won't work anymore. Should we omit them?

Comment (chshersh) [reply CONTRIBUTING.md:13]: Let's keep them for now and see whether they work. If not -- we will rephrase :slightly_smiling_face:

**Comment by vrom911** [comment]
image

It works! (I experimented in `hakyll-shortcut-links` repo)


EDIT:
And in the new project
image

**Pull request merged**